WordPress/Docs/Arrays/ArrayIndentationStandard.xml

```diff
@@ -0,0 +1,52 @@
+<documentation title="Array Indentation">
+    <standard>
+    <![CDATA[
+      The array closing bracket and indentation should line up with the start of the content on the line containing the array opener.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Closing bracket lined up correctly">
+        <![CDATA[
+$args = array(
+    'post_id' => 22,
+<em>);</em>
+        ]]>
+        </code>
+        <code title="Invalid: Closing bracket lined up incorrectly">
+        <![CDATA[
+$args = array(
+            'post_id' => 22,
+        <em>);</em>
+        ]]>
+        </code>
+    </code_comparison>
+     <standard>
+    <![CDATA[
+      Array indentation should follow logical structure
```

Comment:
I think a bit more description is needed here - what counts as "logical" to one person, isn't to someone else.

Something along the lines of: `Keys in each array and nested array should be indented by a 4-space tab for each level of nested array, so that the array visually matches its structure.`

Reply:
It was hard for me to explain this part, so I copied the text from https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

I do agree with your argument, would you mind if I were to use that exact explanation?

Reply:
Go for it :-)

Reply:
Thanks, I've updated the text.